lab_2_tokenize_by_bpe/main.py

```diff
@@ -47,7 +79,19 @@ def merge_tokens(
     :param pair: a pair of tokens to be merged
     :return: dictionary in the form of <preprocessed word: number of occurrences>
     """
-
+    if not isinstance(word_frequencies, dict) or not isinstance(pair, tuple):
+        return None
+    merge_token_dictionary = {}
+    for token in word_frequencies:
+        token_list = list(token)
+        if str(pair)[0 : 2] in str(token):
```

Comment:
why such slicing?

Reply:
I tried to take both two elements but I changed it and now it's done another way